scipy/fftpack/basic.py

```diff
@@ -378,6 +378,19 @@ def fft2(x, shape=None, axes=(-2,-1), overwrite_x=False):
     --------
     fftn : for detailed information.
 
+    Examples
+    --------
+    >>> from scipy.fftpack import fft2, ifft2
+    >>> import numpy as np
+    >>> y = np.mgrid[:5, :5][0]
+    >>> y
+    array([[0, 0, 0, 0, 0],
```

Comment:
I'm not sure why this output is failing in CI, but the failure is real. Perhaps a special character that came in with copy-pasting? You should be able to reproduce the failure locally by running `python tools/refguide_check.py`